app/controllers/home_controller.rb

```diff
@@ -63,6 +63,20 @@ def research
     end
   end
 
+  # Display all posts assuming all posts == activity
+  def all_posts
+    if current_user
+      # underscore used for variables not needed
+      # I'm assuming all comments == @comments + @answer_comments
+      # TO DO: Identify what will be pinned at the top of the page. A featured post maybe?
```

Comment:
I think this TODO is powered by some config maybe as it keeps changes. @jywarren  can confirm

Reply:
I believe this page would not have anything pinned. The dashboard itself would! Thanks!